ludwig/schema/common_fields.py

```diff
@@ -38,12 +38,18 @@ def ResidualField(
     )
 
 
-def NumFCLayersField(default: int = 0, description: str = None, parameter_metadata: ParameterMetadata = None) -> Field:
+def NumFCLayersField(
+    default: int = 0, description: str = None, parameter_metadata: ParameterMetadata = None, non_zero=False
+) -> Field:
     description = description or "Number of stacked fully connected layers to apply."
     full_description = description + (
         " Increasing layers adds capacity to the model, enabling it to learn more complex feature interactions."
     )
     parameter_metadata = parameter_metadata or COMMON_METADATA["num_fc_layers"]
+    if non_zero:
```

Comment:
Could you add a comment about when it's appropriate to support `NumFCLayers=0`?

Reply:
+1